### PR TITLE
[ADD] project_issue_sheet_extended

### PR DESCRIPTION
**Pull request by vrenaville**

### CODE REVIEW

**Comment by jgrandguillaume** [comment]
Hi,

Functional test :+1: 

Thanks for the contrib,

File: project_issue_sheet_extended/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    :alt: License: AGPL-3
+
+Project issue sheet extended
+============================
+
+Add ability to timesheet on issue in the detail tab of the timesheet
+il also add the possibility to timesheet on a child analytic account of
+an account linked with a project.
+
+
```
Comment (rafaelbn) [inline project_issue_sheet_extended/README.rst:11]: Add Installation, configuration and usage section as described here https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by rafaelbn** [comment]
Please @vrenaville when making a PR add a description to it in order to know about what is it. Thanks
I will test it when configuration and usage is described in readme.

File: project_issue_sheet_extended/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+     'name': 'Projects Issue extensions',
```
Comment (dreispt) [inline project_issue_sheet_extended/__openerp__.py:21]: "Projects Issue Time Sheet Extension"

File: project_issue_sheet_extended/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    'version': '1.0',
```
Comment (dreispt) [inline project_issue_sheet_extended/__openerp__.py:22]: "8.0.1.0.0"

File: project_issue_sheet_extended/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    'category': 'Project Management',
+    'summary': 'Add more filed',
```
Comment (dreispt) [inline project_issue_sheet_extended/__openerp__.py:24]: Needs a better one-line summary

File: project_issue_sheet_extended/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    'description': """\
```
Comment (dreispt) [inline project_issue_sheet_extended/__openerp__.py:25]: remove description key

File: project_issue_sheet_extended/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+""",
+    'author': "Daniel Reis,Odoo Community Association (OCA)",
```
Comment (dreispt) [inline project_issue_sheet_extended/__openerp__.py:27]: Are you sure about the author? :wink:

File: project_issue_sheet_extended/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    'license': 'AGPL-3',
+    'depends': [
+        'project_issue',
```
Comment (dreispt) [inline project_issue_sheet_extended/__openerp__.py:30]: 'project_issue' is not necessary

File: project_issue_sheet_extended/project_issue.py [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+        else:
+            custom_values = custom_values.copy()
+        custom_values.setdefault('confirmation_email_sent', False)
+
+        return super(project_issue, self).message_new(
+            self.env.cr, self.env.uid, msg,
+            custom_values=custom_values, context=self.env.context)
+
+    @api.multi
+    def name_get(self):
```
Comment (dreispt) [inline project_issue_sheet_extended/project_issue.py:50]: The change does not belong in this module.

**Comment by pedrobaeza** [comment]
Please rebase to start from a green branch

**Comment by dreispt** [comment]
@vrenaville will you be working on this or should it be closed?

**Comment by dreispt** [comment]
Should we close this?

**Comment by emagdalenaC2i** [comment]
This PR has not changed for more than 2 years. @OCA/project-service-maintainers Should it be closed?

**Comment by simahawk** [comment]
@vrenaville feel free to reopen if needed :wink:

**Pull request closed without merging**